lib/fs.js

```diff
@@ -226,12 +225,13 @@ fs.existsSync = function(path) {
 fs.readFile = function(path, options, callback_) {
   var callback = maybeCallback(arguments[arguments.length - 1]);
 
-  if (!options || typeof options === 'function')
+  if (!options || typeof options === 'function') {
     options = { encoding: null, flag: 'r' };
-  else if (typeof options === 'string')
+  } else if (typeof options === 'string') {
     options = { encoding: options, flag: 'r' };
-  else if (typeof options !== 'object')
-    throw new TypeError('Bad arguments');
+  } else if (typeof options !== 'object') {
+    throwOptionsError(options);
+  }
```

Comment:
@trevnorris Yup, when I was preparing the patch, I am not sure why, but linter was complaining about `throwOptionsError` being in that position. I had  to use `{}` to fix it. Now, linter is fine without `{}`. Shall I revert it?